flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/calcite/FlinkPlannerImpl.scala

```diff
@@ -182,6 +182,18 @@ class FlinkPlannerImpl(
         case compileAndExecute: SqlCompileAndExecutePlan =>
           compileAndExecute.setOperand(0, validate(compileAndExecute.getOperandList.get(0)))
           compileAndExecute
+        // for call procedure statement
+        case sqlCallNode
```

Comment:
nit: Can we use sqlNode.getKind here ?

Reply:
Good suggestion. We can use get kind to make it more clear